scripts/bin/hugo

```diff
@@ -22,7 +22,7 @@ set -o nounset
 set -o pipefail
 
 REPO_ROOT="${REPO_ROOT:-$(cd "$(dirname "$0")/../.." && pwd)}"
-VERSION="0.59.1"
+VERSION="0.84.0"
```

Comment:
https://github.com/cert-manager/website/pull/508#issuecomment-818501996

Reply:
See https://github.com/cert-manager/website/pull/628#discussion_r657915199

Reply:
Yeah, but @maelvls 's concern is that all the anchors remain stable. 
His example still works: https://deploy-preview-628--cert-manager.netlify.app/docs/contributing/building/#but-i-like-makefiles-more

But are there any that have changed? @maelvls You have a script to check this, right?